api/samples/CMakeLists.txt

```diff
@@ -212,7 +227,13 @@ add_sample_client(empty       "empty.c"         "")
 add_sample_client(memtrace_simple "memtrace_simple.c;utils.c" "drmgr;drreg;drutil;drx")
 add_sample_client(memval_simple   "memval_simple.c;utils.c"   "drmgr;drreg;drutil;drx")
 add_sample_client(instrace_simple "instrace_simple.c;utils.c" "drmgr;drreg;drx")
-add_sample_client(opcode_count    "opcode_count.cpp"    "drmgr;drreg;drx;droption")
+# FIXME i#4326: We currently only support additional test for the opcode_count sample on
+# X86 32-bit UNIX. We should port to other archs.
+if (X86 AND (NOT X64) AND UNIX)
+  add_sample_client(opcode_count    "opcode_count.cpp"    "drmgr;drreg;drx;droption")
```

Comment:
These are now identical and this if() would go away?

Reply:
Done.